.test/run.sh

```diff
@@ -608,6 +608,53 @@ fi
 	;&
 	password_hash)
```

Comment:
^ the point of these is so you can start the test suite at specific location. So the value here should be someting like `replication)`.

And put the `;&' and `password_hash)` at the end before those tests.

Reply:
Done 👍🏽